pkgs/development/python-modules/agentops/default.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  setuptools,
+  fetchFromGitHub,
+  pythonOlder,
+  gitUpdater,
+
+  # dependencies
+  requests,
+  psutil,
+  packaging,
+  termcolor,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "agentops";
+  version = "0.3.18";
+  pyproject = true;
+  disabled = pythonOlder "3.7";
+  src = fetchFromGitHub {
+    owner = "AgentOps-AI";
+    repo = "agentops";
+    rev = "refs/tags/${version}";
+    hash = "sha256-g3gKvCNGdeGyKCyTEG2Gss+lI5bGqkKftn8T4bB4c0c=";
+  };
+  passthru.updateScript = gitUpdater { };
+  pythonRelaxDeps = [
+    "psutil"
+    "packaging"
+  ];
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ setuptools ];
+  dependencies = [
+    requests
+    psutil
+    packaging
+    termcolor
+    pyyaml
+  ];
+  doCheck = true;
+  pythonImportsCheck = [ "agentops" ];
+  meta = {
+    description = "AgentOps helps developers build, evaluate, and monitor AI agents.";
+    downloadPage = "https://github.com/AgentOps-AI/agentops/releases";
```

```suggestion
    changelog = "https://github.com/AgentOps-AI/agentops/releases/tag/${version}";
```

Please edit all `downloadPage` entries to be like this.

---

You've changed `downloadPage` to `changelog`. Was that deliberate? I'm going to assume not. Will change the `downloadPage` entries as you suggest without changing them to be `changelog` entries.